static/app/views/settings/components/dataScrubbing/modals/form/selectField.tsx

```diff
@@ -32,13 +32,6 @@ class SelectField extends Component<Props> {
           ...opt,
           details: opt.description ? `(${opt.description})` : undefined,
         }))}
-        styles={{
-          control: (provided: {[x: string]: string | number | boolean}) => ({
-            ...provided,
-            minHeight: '41px',
-            height: '41px',
-          }),
-        }}
         ref={this.selectRef}
         openOnFocus
```

Comment:
Note sure why we needed `41px` here
<img width="600" alt="Credit card numbers" src="https://user-images.githubusercontent.com/44172267/196279308-76b2ab09-333e-4136-9cb5-7bdbb427fed2.png">